test/test_relayselector.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Philipp Winter <phw@nymity.ch>
+#
+# This file is part of exitmap.
+#
+# exitmap is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# exitmap is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with exitmap.  If not, see <http://www.gnu.org/licenses/>.
+""" Unit tests for the relay selector module."""
+
+import unittest
+import sys
+sys.path.insert(0, 'src/')
+import relayselector
+
+
+class TestRelaySelector(unittest.TestCase):
+    """Test the torsocks module."""
+
+    def test_get_exits(self):
+        self.assertRaises(AssertionError, relayselector.get_exits, '/tmp',
+                          'at', True, True, None, None, None, [])
```

Comment:
I would simply omit all optional arguments that you don't modify, i.e., turning it into:
`self.assertRaises(AssertionError, relayselector.get_exits, '/tmp', 'at', True, True)`
That way, it's easier to read what you are doing here